weather_mv/setup.py

```diff
@@ -135,8 +126,7 @@ def RunCustomCommand(self, command_list):
                 'Command %s failed: exit code: %s' % (command_list, p.returncode))
 
     def run(self):
-        for command in CUSTOM_COMMANDS:
-            self.RunCustomCommand(command)
+        pass
```

Comment:
No need to do this now, but I think we could delete the whole `CustomCommands` bit of code.

Reply:
on it

Reply:
done